include/RED4ext/RTTISystem.hpp

```diff
@@ -33,6 +36,7 @@ struct IRTTISystem
     virtual void sub_88() = 0;
     virtual void sub_90() = 0;
     virtual void RegisterFunction(CGlobalFunction* aFunc) = 0;
+    virtual void RegisterProperty(CProperty* aFunc) = 0;
```

Comment:
No clue if RegisterFunction is native etc. I have a suspicion if it is then this should also be but it is literally registering a function